models/git_diff.go

```diff
@@ -164,11 +163,11 @@ func ParsePatch(pid int64, maxlines int, cmd *exec.Cmd, reader io.Reader) (*Diff
 		// Get new file.
 		if strings.HasPrefix(line, DIFF_HEAD) {
 			if isTooLong {
-				return diff, nil
+				break
 			}
 
-			fs := strings.Split(line[len(DIFF_HEAD):], " ")
-			a := fs[0]
+			beg := len(DIFF_HEAD)
+			a := line[beg : (len(line)-beg)/2+beg]
```

Comment:
Just to confirm, is it guess based on assumption that length of two files are equal?